straxen/plugins/peaks/peak_basics.py

```diff
@@ -15,40 +15,10 @@ class PeakBasics(strax.Plugin):
 
     """
 
-    __version__ = "0.1.4"
+    __version__ = "0.1.5"
```

Comment:
Presumably this is also just from PR #1299 ?

Reply:
Yes, same as above purely optional.